ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -470,22 +506,42 @@ <h3 class="title">
           </div>
         </th>
       </ng-container>
+
+      <ng-container matColumnDef="srcIndexColName">
```

Comment:
deleting a column from index in spanner is deleting it from source too in UI

Reply:
If there are multiple columns in an index, then column can be drop without dropping the index altogether. But if the index have only one column then dropping that column will result in dropping the index. This is how the current UI works as of now.

Reply:
Could you please elaborate a bit more on this issue?

Reply:
deleting a column from an index in UI should not delete that column from source tab in index

Reply:
Resolved